crates/typst-cli/src/main.rs

```diff
@@ -37,17 +38,25 @@ fn main() -> ExitCode {
     let timer = Timer::new(&ARGS);
 
     let res = match &ARGS.command {
```

Comment:
maybe we could move the match into a separate function, so that we can use `?` instead of all the `map_err`? could be called `run` or `dispatch`.